raster-foundry/raster-foundry
# Use shapes for AOIs

## Overview
Update the backend to use shapes instead of an area field
Update the frontend to create shapes for AOIs

### Checklist

- ~Styleguide updated, if necessary~
- [x] [Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated https://github.com/raster-foundry/raster-foundry-api-spec/pull/38
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~

### Notes
I haven't added selecting existing shapes to the aoi parameter edit page yet, since I don't think there's a design for it.

## Testing Instructions

 * Make an AOI before switching to this PR's branch
* Verify that it works (run the aoi update locally using `rf update-aoi-project {projectid}` in the batch container `./scripts/console batch bash`
* Switch to this branch and run migrations
* Verify that the AOI still works 
* Make a new AOI and verify it works
* Edit an AOI and verify that it works
Closes #3607
